### DISCUSSION TITLE
Issue with the trigger value on alert recovery

### DISCUSSION THREAD

**Discussion by hdsfgsgrlsgtw12** [ccfos/nightingale#1385] [Q&A]
麻烦问一下哈  触发时值是.TriggerValue定义的，是告警时候触发的值，如下：
![image](https://user-images.githubusercontent.com/45323722/220031540-56149aaf-01b7-4c13-aea0-7968895d2699.png)

但是恢复的话只有恢复信息，并没有恢复时候的值，请问下恢复时值是哪个字段来定义的呢？比如磁盘告警的时候我可以通过TriggerValue知道磁盘使用多少了 ，那我清理磁盘之后恢复了，值应该也要告警告出来的，我要清楚我磁盘还有多少空间可用
![image](https://user-images.githubusercontent.com/45323722/220031598-3e2866d9-ddec-4ccf-8d80-8cab91527294.png)

**Comment by 710leo** [comment]
可以参考下这个 https://github.com/ccfos/nightingale/issues/1135
